packages/jaeger-ui/src/model/trace-dag/types/TDagPlexusVertex.tsx

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { TVertex } from '@jaegertracing/plexus/lib/types';
+import { TVertex } from '@jaegertracing/plexus/src/types';
```

Comment:
I am actually somewhat skeptical about this - all the imports you fixed are referring to types. Types don't contain any functionality, so while useful when the types themselves are changing, they won't fix the actual hot reloading issue.

Also, see my other comment https://github.com/jaegertracing/jaeger-ui/pull/2089/files#r1443119010. I think if you find a way to instruct jaeger-ui package at a global level to prefer src instead of lib of Plexus, you may not need to change these imports.

Reply:
Yes, but even if it is changed at global level. The individual import will specifically still point to the lib instead of src right? Correct me if I am wrong?

Reply:
you have to look into those suggestions from Bard - my understanding is if you override at a global level then the build system will dynamically replace the imports

Reply:
Understood!

Reply:
Hi, @yurishkuro have been working on this since yesterday night. Tried various methods, and on modifying the tsconfig.json file, to point @jaegertracing/plexus to src package all the imports now automatically point to plexus/src now.
![Screenshot 2024-01-06 195739](https://github.com/jaegertracing/jaeger-ui/assets/103821431/aeb7719b-c3a8-402f-89ed-49affa530489)
But, the problem that is not being resolved and I am stuck at is that Hot reload is still not working even after changing the address. Do you have any input that could be causing this?

Reply:
It is really confusing, just manually changing the import statement to '@jaegertracing/plexus/src' is making the Hot Reload work completely fine, even though it points to the exactly same directory on hovering.
![Screenshot 2024-01-06 201211](https://github.com/jaegertracing/jaeger-ui/assets/103821431/245dc120-af9e-48e2-9340-7207c7efb51e)
Hot reload is working perfectly fine in this.